.vscode/launch.json

```diff
@@ -37,10 +28,7 @@
       "request": "launch",
       "module": "pytest",
       "justMyCode": false,
-      "args": [
-        "--timeout=10",
-        "--picked"
-      ],
+      "args": ["--timeout=10", "--picked"]
```

Comment:
[same](https://github.com/home-assistant/core/pull/127042/files#r1786851644) happening here, whenever I hit save something autolints short arrays to be on one line